modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM710AllSubscriptionsByApplicationTestCase.java

```diff
@@ -237,7 +222,7 @@ public void setEnvironment() throws Exception{
             Thread.sleep(3000);
```

Comment:
This file has several sleeps in between loops. The sleeps are here NOT to wait until getting the response, but rather give delay between sending requests to APIM. APIM should be able to handle multiple requests parallel. So, why do we need sleeps here?

Reply:
Fixed with 4f6054ea9733fd040759435c990d6a53cfb56c7a